scanner/src/main/kotlin/utils/ScanCodeResultParser.kt

```diff
@@ -20,7 +20,7 @@
 
 @file:Suppress("TooManyFunctions")
 
-package org.ossreviewtoolkit.scanner.scanners
+package org.ossreviewtoolkit.scanner.utils
```

Comment:
Not sure whether this move has that much benefit. There is now a cyclic dependency between the _scanners_ and the _utils_ packages, so they are not really independent. As the ScanCode implementation is complex, would it make sense instead to move the _ScanCode_ and _ScanCodeResultParser_ classes into a new sub package _scanners.scancode_?

Reply:
I like that idea, esp. in the context of [ScanOSS integration](https://github.com/oss-review-toolkit/ort/pull/3278) which also introduces its own utility functions; moving the utility functions into scanner-specific packages allows to name the functions the name without creating name clashes.